qemu/qemu.go

```diff
@@ -1456,8 +1456,8 @@ type Knobs struct {
 	// be set.
 	FileBackedMem bool
 
-	// FileBackedMemShared will set the FileBackedMem device as shared.
-	FileBackedMemShared bool
+	// MemShared will set the FileBackedMem device as shared.
```

Comment:
Is this comment still correct as MemShared now applies to other backends and not just FileBackedMem?

Reply:
oops, fixed. Thanks!